Makefile

```diff
@@ -76,7 +76,34 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
-ifeq ($(filter $(app),apisix dashboard),)
+### build apisix-openresty:
+.PHONY: build-apisix-openresty-rpm
+build-apisix-openresty-rpm:
+	mkdir -p ${PWD}/build/rpm
+	docker build -t apache/apisix-openresty:$(version) --build-arg checkout_v=$(checkout) -f ./dockerfiles/Dockerfile.apisix-openresty.rpm .
```

Comment:
Do we need both `version` and `checkout`? They look the same.

Reply:
I was thinking of using `checkout` to control the version of the source openresty and `version` to control the version of the image that is build